README.md

```diff
@@ -0,0 +1,47 @@
+<h1 align="center">
+    High-quality, ubiquitous, and portable telemetry to
+    enable effective observability
+</h1>
+
+<p align="center">
+  <a href="https://opentelemetry.io/">
+    <img src="https://opentelemetry.io/img/logos/opentelemetry-horizontal-color.svg" alt="OpenTelemetry logo" height="180">
+  </a>
+</p>
+
+<p align="center">
+OpenTelemetry is a collection of tools, APIs, and SDKs. Use it to
+instrument, generate, collect, and export telemetry data (<a href="https://opentelemetry.io/docs/concepts/signals/metrics/">metrics</a>, <a href="https://opentelemetry.io/docs/concepts/signals/logs/">logs</a> and <a href="https://opentelemetry.io/docs/concepts/signals/traces/">traces</a>) to help you analyze your software’s performance
+and behavior.
+</p>
+
+<p align="center">
+  <a href="https://opentelemetry.io/docs/what-is-opentelemetry/">Learn More 📚</a>
+  ·
+  <a href="https://opentelemetry.io/docs/getting-started/">Get Started 🔭</a>
+  ·
+  <a href="https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md">Contribute 🫶</a>
```

Comment:
Something is wrong with the emoji here: `Contribute 🫶`

Reply:
@pellared it works for me (on MacOS), can you share a screenshot of what you are seeing? I can take another one

Reply:
![image](https://github.com/open-telemetry/.github/assets/5067549/0ad82d98-bf7b-4c1f-8db0-c21b31fd513d)

(Windows 10)

Reply:
oh... thanks, I changed it to a different one, hope that one works :)

Reply:
works 👍